src/unity/paths/accounts_accountId.yml

```diff
@@ -24,7 +50,7 @@ patch:
       content:
         application/json:
           schema:
-            $ref: "../schemas/Account.yml"
+            $ref: "../schemas/UserAccount.yml"
```

Comment:
I realize that this might be a potential breaking change in the UI, but would it be better to have this stay as the new `Account` schema for consistency instead?

It would mean we could avoid doing extra work in the `PATCH` controller.

On further thought, renaming schemas is already a "breaking" UI change, so maybe we could do this at the same time?

If you do make this change, can you add to the A/C of #6205 to remind us to actually do the simplification there?

Reply:
Are you saying that we should update the Quartz controller in this story as we make the change, or as part of #6205?

Reply:
I was thinking as part of #6205, but that might not make sense because the controller won't be returning the `type` and `billingProvider` fields yet.

I think that continuing to return the (soon to be) extra `isActive` and `isDefault` fields is fine until we do the controller work, but not returning the other two fields might confuse someone.

That said, I now see below that those two fields are not marked as `required` so maybe it's OK?

Reply:
It seems that this change doesn't fit well here or in #6205, why don't I make a separate story to update the PATCH call in the controller and openapi to use the new `Account` schema?

Reply:
Works for me.